data/migrations/20200204164931_cars-schema.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('cars', tbl =>{
+      tbl.increments();
+      tbl.text('Make', 128).unique().notNullable();
+      tbl.text('Model', 128).unique().notNullable();
+      tbl.decimal('year');   
```

Comment:
year is not needed, i would recommend using an integer instead of a decimal in any case for numbers